packages/dd-trace/test/exporters/common/request.spec.js

```diff
@@ -0,0 +1,164 @@
+'use strict'
```

Comment:
almost the same file as `packages/dd-trace/test/exporters/agent/request.spec.js`, with some minor tweaks